packages/frontend/src/components/shifts/ShiftStack.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import Schedule from 'backend/models/schedule/schedule';
+import { Stack } from '@mui/material';
+import Shift from 'backend/models/shift/shift';
+import BackendShiftClient from 'src/api/shifts/shifts';
+import { getConfig } from 'backend/config/config';
+import { ShiftBlock } from './ShiftBlock';
+
+interface Props {
+  schedule: Schedule;
+  timeSlots: Date[];
+}
+
+export default function ShiftStack(props: Props) {
+  const appConfig = getConfig();
+  const shiftClient = new BackendShiftClient(appConfig.BackendURL);
```

Comment:
Same comment here about memoized the client and pulling the `getConfig` outside of the component

Reply:
Done